spl/src/metadata.rs

```diff
@@ -562,7 +562,18 @@ pub struct BurnEditionNft<'info> {
     pub edition_marker: AccountInfo<'info>,
     pub spl_token: AccountInfo<'info>,
 }
-
+/// BurnNft instruction takes in an optional 'collection_metadata' 'PubKey'.
```

Comment:
Could you use backticks when referencing code like `collection_metadata` instead of quotes (')?

Reply:
done.